workbook/01-introduction.asciidoc

```diff
@@ -105,7 +105,7 @@ More than one answer may be correct in some questions.
 
 ====== Correct answer: 1
 
-Why 1 is correct: The host teams open their code base to others and put effort into vetting contributions precisely because their code end up better and more featureful than if they did all the coding themselves.
```

Comment:
This is my favorite correction ... `featureful` 😜.

Reply:
Hehe yeah, I saw this one too.
I think `featureful` should be a real word :)